CopilotKit/packages/shared/src/utils/random-id.ts

```diff
@@ -0,0 +1,13 @@
+export function randomId(length = 16, alphabet = "abcdefghijklmnopqrstuvwxyz0123456789") {
```

Comment:
We can instead simply use `globalThis.crypto.randomUUID()`

Reply:
Done but we keep the `@copilotkit/shared` dependency